src/lib/github.js

```diff
@@ -1,21 +1,55 @@
-const client = require("./client");
-const { fgGreen, fgRed } = require("../utils/color");
+const { fgGreen, fgRed, reset } = require("../utils/color");
+const { createBrowserInstance } = require("./browser");
 
-module.exports = async (repository, word) => {
-  const url = `${repository}/search?q=${word}&type=code`;
+function githubClient() {
+  const validGithubRepositoryUrlRegex = /https:\/\/(www\.)?github\.com\/.*\/.+/;
```

Comment:
This constant is part of the validateUrl() function, it doesn't make sense to declare it here.

Reply:
So every time we need to call `validateUrl` this constant will be created? In my opinion, this constant should be outside of githubClient.

Reply:
Yes, it would be much more performant for this constant to be created only when executing the `validateUrl` function and not leave it created outside its scope and be created whenever the `githubClient` is called.

Reply:
Moved to out of githubClient scope, which makes more sense. If you don't like this, just comment here and I will change.